nomad/provider_test.go

```diff
@@ -29,16 +30,38 @@ func TestProvider(t *testing.T) {
 
 var testProvider *schema.Provider
 var testProviders map[string]terraform.ResourceProvider
+var managementToken string
 
 func init() {
 	testProvider = Provider().(*schema.Provider)
 	testProviders = map[string]terraform.ResourceProvider{
 		"nomad": testProvider,
 	}
+
+	if v := os.Getenv("TF_ACC"); v == "1" {
+		setupAcl()
+	}
 }
 
 func testAccPreCheck(t *testing.T) {
 	if v := os.Getenv("NOMAD_ADDR"); v == "" {
 		os.Setenv("NOMAD_ADDR", "http://127.0.0.1:4646")
 	}
+	os.Setenv("NOMAD_TOKEN", managementToken)
+}
+
+func testAccPreCheckAnonymous(t *testing.T) {
+	if v := os.Getenv("NOMAD_ADDR"); v == "" {
+		os.Setenv("NOMAD_ADDR", "http://127.0.0.1:4646")
+	}
+	os.Unsetenv("NOMAD_TOKEN")
+}
+
+func setupAcl() {
+	client, _ := api.NewClient(&api.Config{})
```

Comment:
It looks like we're ignoring an error here. I'd be a lot more comfortable if we checked and panicked on it.